src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -299,21 +302,22 @@ public static void registerConfigCondition(String modId, Object... configObjects
 				}
 			}
 		}
-		CraftingHelper.register(new ConfigValueCondition.Serializer(modId, configValues));
+		CraftingHelper.register(new ConfigCondition.Serializer(modId, configValues));
+		Registry.register(Registry.LOOT_CONDITION_TYPE, new ResourceLocation(AbnormalsCore.MODID, "config"), new LootConditionType(new ConfigLootCondition.Serializer(modId, configValues)));
```

Comment:
Shouldn't this use `modId`, not `AbnormalsCore.MODID`?

Reply:
...yes.

Reply:
Fixed.